src/js.rs

```diff
@@ -148,6 +148,12 @@ extern "C" {
     #[wasm_bindgen(method)]
     pub fn find(this: &Array, predicate: &mut FnMut(JsValue, u32, Array) -> bool) -> JsValue;
 
+    /// The forEach() method executes a provided function once for each array element.
+    ///
+    /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach
+    #[wasm_bindgen(method, js_name= forEach)]
```

Comment:
space between `js_name` and `forEach` like so? `js_name = forEach)]`

Reply:
Good catch, just fixed.